src/__init__.py

```diff
@@ -20992,6 +21040,36 @@ def set_graphics_min_line_width(min_line_width):
         '''
         mupdf.fz_set_graphics_min_line_width(min_line_width)
 
+
+    @staticmethod
+    def get_word_delimiters():
+        """Return extra word delimiting characters."""
+        global g_word_delimiters
+        delims = [chr(c) for c in g_word_delimiters if c != 0]
+        return delims
+
+
+    @staticmethod
+    def set_word_delimiters(delims=None):
+        """Set extra word delimiting characters."""
+        global g_word_delimiters
+        if delims is None:
+            delims = []
+        if not hasattr(delims, "__getitem__") or len(delims) > 64:
+            raise ValueError("bad delimiter value(s)")
+        try:
+            delims = set([ord(c) for c in delims if ord(c) > 32])
+        except:
+            print("bad delimiter value(s)")
+            raise
+        delims = tuple(delims)
+        if not delims:
+            g_word_delimiters = [0] * 65
+        else:
+            g_word_delimiters = delims
+        return # extra.set_word_delimiters(delims)
```

Comment:
If we can set `g_word_delimiters` to a set here, we probably don't need it to be `[0]*65` when we initiallise?